site/docs/messaging/quickStart.mdx

```diff
@@ -0,0 +1,77 @@
+---
+id: quickStart
+title: Quick Start Guide
+slug: /messaging/quickStart
+description: A Quick Start Guide for using the Messaging API
+keywords:
+  - bandwidth
+  - messaging
+  - guide
+  - tutorial
+  - quick
+  - start
+hide_title: false
+image: ../../static/img/bandwidth-logo.png
+---
+
+This is a brief walkthrough for setting up and using a Bandwidth messaging application. This guide will outline sending a message, receiving a message, and receiving status callbacks.
+
+## Assumptions
+* You have already created and set up your [Bandwidth Dashboard](https://dashboard.bandwidth.com) account. 
+    * If you have not already done so, please look [here](/docs/account/setup) for our Account Setup guide.
+* You have created a messaging application in the Dashboard UI or using the Dashboard API. 
+    * Instructions for doing so can be found [here](/docs/account/applications).
+
+With an account and application created, we can get started using the Messaging API.
+
+## In This Guide
+* [Configuring Callback URLs](#configuring-callback-urls)
+    * [Creating a Personal Server](#creating-a-personal-server)
+    * [Using Pipedream](#using-pipedream)
+* [Sending a Message](#sending-a-message)
+    * [Sent Message Callbacks](#sent-message-callbacks)
+* [Receiving a Message](#receiving-a-message)
+
```

Comment:
```suggestion
## In This Guide
* [Configuring Callback URLs](#configuring-callback-urls)
    * [Creating a Personal Server](#creating-a-personal-server)
    * [Using Pipedream](#using-pipedream)
* [Sending a Message](#sending-a-message)
    * [Sent Message Callbacks](#sent-message-callbacks)
* [Receiving a Message](#receiving-a-message)

```
I believe we can likely nuke this section since Docusaurus tosses this info on the right of the page automatically.

Reply:
@ajrice6713 thoughts?

Reply:
I concur - i think I got rid of most of these when migrating the other guides in favor of the right hand side TOC - one less thing to maintain

Reply:
didnt even notice it did that, I'll take this out